packages/MJExplorer/src/app/app.component.ts

```diff
@@ -107,6 +107,8 @@ export class AppComponent implements OnInit {
       }
     }, (err: any) => {
       LogError('Error Logging In: ' + err);
+      //show the login page
+      this.authBase.authenticated = false;
```

Comment:
Not a problem, but shouldn't the authBase object be setting that value itself when it hits an error in the `getUseClaims` method?

Reply:
That's a better place to put it, yea. I've applied the changes.